pkgs/top-level/config.nix

```diff
@@ -61,7 +61,14 @@ let
 
     allowAliases = mkOption {
       type = types.bool;
-      default = true;
+      default =
+        let
+          envVar = builtins.getEnv "NIXPKGS_ALLOW_ALIASES";
+        in
+        if envVar != ""
+        then envVar != "0"
+        else true;
+      defaultText = literalMD ''`true` unless `getEnv "NIXPKGS_ALLOW_ALIASES" == "0"`'';
```

Comment:
IMO we should use some library function which accepts 0, 1, true, false, TRUE, True, FALSE and False just to make it easy for everyone.